frontend/src/libs/components/toggle-checkbox/styles.module.scss

```diff
@@ -0,0 +1,49 @@
+.container {
+  display: flex;
+  align-items: center;
+  width: fit-content;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+.switch {
+  position: relative;
+  width: 56px;
+  height: 24px;
+  margin-right: 15px;
+  background-color: rgba(0 0 0 / 5%);
```

Comment:
I believe it is better to create a variable

Reply:
Why ? Do you think we need variable for every possible color-opacity combination ? 
It's not being used anywhere else right now as far as I know

Reply:
I think because of reusing it, and maybe we could use it later, but it's up to you.